libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/DrillTargetType/DrillTargetTypeListItem.tsx

```diff
@@ -37,6 +39,16 @@ const DrillTargetTypeListItem: React.FC<IDrillTargetTypeListItemProps> = ({
             <SingleSelectListItem
                 className={className}
                 title={item.title}
+                info={
+                    formatMessage(
+                        { id: item.tooltipMessage },
+                        {
+                            a: (chunk: ReactNode) => {
+                                return <a href={item.documentUrl}>{chunk}</a>;
+                            },
+                        },
+                    ) as string
```

Comment:
May I ask for change this ListItem component to accept JSX element instead of cast here?